spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -508,11 +508,11 @@ class CometExecSuite extends CometTestBase {
           }.map(_.metrics).get
 
           assert(metrics.contains("input_batches"))
-          assert(metrics("input_batches").value == 8L)
+          assert(metrics("input_batches").value == 2L)
           assert(metrics.contains("input_rows"))
           assert(metrics("input_rows").value == 10L)
           assert(metrics.contains("output_batches"))
-          assert(metrics("output_batches").value == 4L)
+          assert(metrics("output_batches").value == 1L)
```

Comment:
The metrics are changed due to https://github.com/apache/datafusion-comet/pull/553/files#r1730694991.